Gemfile

```diff
@@ -29,6 +29,7 @@ group :development do
   gem "ruby-prof", :platform => :mri
 
   gem "json",      :platform => :ruby_18
+  gem "celluloid-io"
```

Comment:
Sockets won't convert to celluloid-io's sockets unless you're using the reactor.
I'll make the celluloid tests optional. I'm still working on them.